server/src/main/java/io/unitycatalog/server/service/TemporaryVolumeCredentialsService.java

```diff
@@ -25,17 +30,7 @@ public HttpResponse generateTemporaryTableCredential(
       throw new BaseException(ErrorCode.INVALID_ARGUMENT, "Volume ID is required.");
     }
     VolumeInfo volumeInfo = VOLUME_REPOSITORY.getVolumeById(volumeId);
```

Comment:
Same question, does this throw if not exist?

Reply:
nm I see the check in `vendCredentialForVolume()`

Reply:
same as above